src/sizzle.js

```diff
@@ -846,6 +846,12 @@ setDocument = Sizzle.setDocument = function( node ) {
 				rbuggyQSA.push( ":enabled", ":disabled" );
 			}
 
+			// Support: Firefox <=3.6 - 5 only
+			// Old Firefox doesn't throw on a badly-escaped identifier.
+			el.querySelectorAll( "x\\\f" );
+			rbuggyQSA.push( "\\r", "\\n", "\\f" );
```

Comment:
This would probably be smaller like `rbuggyQSA.push( "[\\r\\n\\f]" )`, matching other regexp patterns involving the characters.

Reply:
Good point, I'll update it.

Reply:
Updated.